src/ensembl/src/content/app/about-ensembl/AboutEnsembl.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import { DocLayout } from 'src/shared/components/layout';
+import docStyles from 'src/shared/components/layout/DocLayout.scss';
+import styles from './AboutEnsembl.scss';
+
+const AboutEnsembl = () => {
+  return (
+    <div className={styles.aboutEnsembl}>
+      <DocLayout
+        appHeaderContent={<AppHeaderContent />}
+        searchBoxContent={<SearchBoxContent />}
+        globalNavContent={<GlobalLinks />}
+        mainContent={<Page />}
+        localNavContent={<LocalLinks />}
+      />
+    </div>
+  );
+};
+
+const AppHeaderContent = () => {
+  return (
+    <>
+      <p className={docStyles.runningHeader}>About Ensembl</p>
+    </>
+  );
+};
+
+const SearchBoxContent = () => {
+  return (
+    <>
+      <input type="text" placeholder="Search About Ensembl" />
+    </>
```

Comment:
This could just be 
`return <input type="text" placeholder="Search About Ensembl" />`